tests/hil-unity-checklist.yaml

```diff
@@ -0,0 +1,35 @@
+Infineon.xmc.XMC4700_Relax_Kit:

```

Comment:
This is used by the hi l.yml worfklow I guess? 
Shall this be in future here or in the arduino-core-tests repo? 
If it refers to CAN and Wire I am not sure if this is longer of this domain...
But okay, let´s not worry to much about this one. Things we need to think about.

Reply:
@OlafFilies